cypress/integration/login_page.spec.js

```diff
@@ -21,32 +23,74 @@ describe("Login Page", () => {
-      }).as("userSuccess");
-      cy.get("input[id='loginEmail']").type("test@cds-snc.ca");
+    it("Displays an error message when submitting an empty username.", () => {
       cy.get("[type='submit']").click();
-      cy.get("[data-testid='alert']").should("be.visible");
+      cy.get("[id='errorMessageusername']").should(
+        "contain",
+        "Please complete the required field to continue"
+      );
+    });
+    it("Displays an error message when submitting an invalid email", () => {
+      cy.get("input[id='username']").type("myemail@cds-snc");
+      cy.get("[type='submit']").click();
+      cy.get("[id='errorMessageusername']").should("contain", "Please enter a valid email");
+    });
+    it("Displays no error message when submitting a valid email", () => {
+      cy.get("input[id='username']").type("test@cds-snc.ca");
+      cy.get("[type='submit']").click();
+      cy.get("[id='errorMessageusername']").should("not.exist");
     });
-  });
 
-  describe("Temporary Token State", () => {
-    it("Submits good bearer token and displays the temporary token state", () => {
-      cy.visit("/en/auth/login");
-      cy.intercept("POST", "/api/token/temporary", {
-        statusCode: 200,
-      }).as("userSuccess");
-      cy.get("input[id='loginEmail']").type("test@cds-snc.ca");
-      cy.get("textarea[id='signInKey']").type("fAkE_sIgN_iN_kEy");
+    it("Displays an error message when submitting an empty password.", () => {
+      cy.get("[type='submit']").click();
+      cy.get("[id='errorMessagepassword']").should(
+        "contain",
+        "Please complete the required field to continue"
+      );
+    });
+
+    it("Displays an error message when submitting a password greater than 50 characters", () => {
+      cy.get("input[id='password']").type("AAAaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaH");
       cy.get("[type='submit']").click();
+      cy.get("[id='errorMessagepassword']").should(
+        "contain",
+        "This field must be no longer than 50 characters"
+      );
     });
-    it("Displays the temporary token state after a successful response", () => {
-      cy.get("[data-testid='alert']").should("not.exist");
+
+    it("Displays no error message when submitting a valid password", () => {
+      cy.get("input[id='password']").type("AAAaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa");
+      cy.get("[type='submit']").click();
+      cy.get("[id='errorMessagepassword']").should("not.exist");
     });
-    it("Displays an error when entering an invalid temporary token", () => {
+
+    it("Displays an error message when submitting a form and getting an unsuccessful reply from the server.", () => {
+      cy.intercept("POST", "/api/token/temporary", {
+        statusCode: 403,
+      }).as("userSuccess");
+      cy.get("input[id='username']").type("test@cds-snc.ca");
       cy.get("[type='submit']").click();
       cy.get("[data-testid='alert']").should("be.visible");
     });
   });
+
```

Comment:
This test was failing (no loginEmail ID or signInKy ID in the page). I thought it best to comment out for now - even though this is not a BP :)

Reply:
Yup these are old tests. Going to remove for now as they no longer apply